icons/calendar-arrow-down.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "karsa-mistmere",
+    "colebemis",
+    "ericfennis"
+  ],
+  "tags": [
+    "date",
+    "month",
+    "year",
+    "event",
+    "sort",
+    "order",
+    "descending"
```

Comment:
@ericfennis, @jguddas Not sure about these, I followed the tags from earlier sort icons, but to me `arrow-down` would me ascending and `arrow-up` descending. 🤔

Maybe we should add both tags to both, just in case. 🤷‍♂️

Reply:
> Maybe we should add both tags to both, just in case. 🤷‍♂️

I say yes to adding both.

Reply:
> I say yes to adding both.

Done!